ci/.gitlab-ci.yml

```diff
@@ -116,7 +116,7 @@ cpu release build clang13 cxx20:
     DEPLOY_DOCKER_FILE: ci/docker/deploy.Dockerfile
     BASE_IMAGE: docker.io/ubuntu:22.04
     DEPLOY_BASE_IMAGE: docker.io/ubuntu:22.04
-    EXTRA_APTGET: clang-13
+    EXTRA_APTGET: "clang-13 libomp-13-dev"
```

Comment:
We might need to add `libomp` (or a similar package) for the ROCm environment further below to fix this: https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/4700071344751697/7514005670787789/-/jobs/4196109579.